docs/reference/docs/reindex.asciidoc

```diff
@@ -1,6 +1,10 @@
 [[docs-reindex]]
 == Reindex API
 
+IMPORTANT: Reindex requires 
+[`_source`](https://www.elastic.co/guide/en/elasticsearch/reference/6.4/mapping-source-field.html) 
```

Comment:
it should be possible not to use concrete versions in here.  I think we should have a relative link to the _source page using the `mapping-source-field` placeholder, something like `<<mapping-source-field,`_source`>>`. could you make this change please?

Reply:
sorry, forgot about that syntax!